src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1711,6 +1711,16 @@ def InitGroupTestingData(self):
                 self.devCtrl)
         ).raise_on_error()
 
+    def CreateManualCode(self, discriminator: int, passcode: int) -> str:
+        """ Creates a standard flow manual code from the given discriminator and passcode."""
+        # 64 bytes is WAY more than required, but let's be safe
+        size = 64
+        buf = create_string_buffer(size)
+        self._ChipStack.Call(
+            lambda: self._dmLib.pychip_CreateManualCode(discriminator, passcode, buf, size)
```

Comment:
You won't know about how many charactres are in teh code, since there is no output size or null termination. Suggest having an output argument for final size

Reply:
added.